src/commands/demo-config/create.ts

```diff
@@ -0,0 +1,126 @@
+import { getLogger } from "../../logger";
+import { highlight } from "../../utils";
+import { handler as CreateTemporaryDns } from "../dns/txt-record/create";
+import ConfigTemplate from "./config-template.json";
+import { CreateConfigCommand } from "./config.type";
+
+interface ConfigFile {
+  wallet: string;
+  forms: {
+    defaults: {
+      issuers: {
+        name: string;
+        documentStore?: string;
+        tokenRegistry?: string;
+        identityProof: {
+          type: string;
+          location?: string;
+        };
+      }[];
+    };
+  }[];
+}
+
+const { trace } = getLogger("demo-config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a demo config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
+      type: "string",
+      description: "Path to file containing wallet.json",
+      normalize: true,
+      demandOption: false,
+    });
+
+export const handler = async (args: CreateConfigCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  if (!fs.existsSync(args.outputDir)) {
+    fs.mkdirSync(args.outputDir);
+  }
+
+  try {
+    let walletPath = "";
+    if (!args.encryptedWalletPath) {
+      info(`Please enter password to create wallet`);
+      const createWalletParams = {
+        outputFile: `${args.outputDir}/wallet.json`,
+        fund: "ropsten",
+      };
+      walletPath = await CreatedWalletPath(createWalletParams);
+      info(`Wallet created at ${walletPath}`);
+    }
+
+    const walletFile = walletPath ? walletPath : args.encryptedWalletPath;
+    const wallet = await readFile(walletFile);
+
+    info(`Enter password to continue deployment of Document Store`);
+    const deployDocumentStoreParams = {
+      encryptedWalletPath: walletFile,
+      network: "ropsten",
+      gasPriceScale: 1,
+      dryRun: false,
+      storeName: "Demo Document Store",
+    };
+    const documentStore = await deployDocumentStore(deployDocumentStoreParams);
+    const documentStoreAddress = documentStore.contractAddress;
+    success(`Document store deployed, address: ${highlight(documentStoreAddress)}`);
+
+    info(`Enter password to continue deployment of Token Registry`);
+    const deployTokenRegistryParams = {
+      registryName: "Demo Token Registry",
+      registrySymbol: "DTR",
+      encryptedWalletPath: walletFile,
+      network: "ropsten",
+      gasPriceScale: 1,
+      dryRun: false,
+    };
+    const tokenRegistry = await deployTokenRegistry(deployTokenRegistryParams);
+    const tokenRegistryAddress = tokenRegistry.contractAddress;
+    success(`Token registry deployed, address: ${highlight(documentStoreAddress)}`);
+
+    info(`Creating temporary DNS for document store`);
+    const DocumentStoreTemporaryDnsParams = {
```

Comment:
we should run this under condition (if location is dns-txt)

Reply:
updated it to cater for dns-did